package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@frctl/twig",
-  "version": "1.0.0-beta.4",
+  "version": "1.0.0-beta.5",
```

Comment:
I'll check this out as soon as I get a chance, but in the meantime would you mind getting rid of the version bump please? We'll bump the version if/when we make a release so we don't miss a release by mistake.